packages/@guardian/source-react-components/src/hide/Hide.tsx

```diff
@@ -18,12 +18,16 @@ export interface HideProps extends HTMLAttributes<HTMLDivElement>, Props {
 	 * **Deprecated**
 	 *
 	 * Use `from` instead.
+	 *
+	 * @deprecated Use `from` instead.
 	 */
 	above?: Breakpoint;
 	/**
 	 * **Deprecated**
```

Comment:
these one's are for storybook, which ignores the `@deprecated` jsdoc tag

<img width="471" alt="image" src="https://user-images.githubusercontent.com/867233/158824130-ad1072ad-94c6-4fed-832f-4f7e495da44b.png">